src/main/CommandLine.cpp

```diff
@@ -940,6 +962,7 @@ handleCommandLine(int argc, char* const* argv)
          {"upgrade-db", "upgade database schema to current version",
           runUpgradeDB},
 #ifdef BUILD_TESTS
+         {"run-relay", "run relay-mode stellar-core node", runAsRelay},
```

Comment:
I think that in future it may be better to have different NODE_TYPE options in configuration file (VALIDATOR, RELAY, NON-VALIDATOR?), instead of making that a new test-only command.

Reply:
Possible. I'm going to not do this for now unless you feel really strongly; the initialization path is a bit different, we'd need to change config (currently use NODE_IS_VALIDATOR), and I'm not even sure we want to surface relay mode to users yet. Maybe in a followup PR if we decide to keep it / make it a supported mode.

Reply:
No, I don't feel strong about it. That is just maybe-nice-to-have-in-future note.